bgpd/bgp_route.c

```diff
@@ -12347,7 +12347,9 @@ void route_vty_out_detail_header(struct vty *vty, struct bgp *bgp,
 						VRF_DEFAULT_NAME);
 				else
 					vty_out(vty, ", vrf %s",
-						bgp->name);
+						(bgp->inst_type == BGP_INSTANCE_TYPE_DEFAULT)
```

Comment:
This condition is useless, it's checked above, right? What I suggest is replacing these checks at all using `bgp->name_pretty`. In this case no need to check if it's a default instance and/or VRF/view instance.